nullable/bool.go

```diff
@@ -0,0 +1,58 @@
+package nullable
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+	"reflect"
+)
+
+// Bool defines a nullable bool
+type Bool struct {
+	Bool  bool
```

Comment:
Should the internal values be exposed?

Reply:
Yes, as that's how they are accessed.
Unless you want to use getters, but I am thoroughly against that